uaclient/status.py

```diff
@@ -234,6 +234,38 @@ class UserFacingStatus(enum.Enum):
 MESSAGE_REFRESH_FAILURE = "Unable to refresh your subscription"
 
 
+def action_report(
```

Comment:
Adopting new UX generally would involve a bit of review with a broader audience(other teams), instead of adding a new function, we could adopt to maybe  single UserFacingError() with the related misses/invalids and point to running ua status as we did in the current master when someone specifies invalid service names

Reply:
No problem @blackboxsw, I will update the code